src/app/sign-up/page.tsx

```diff
@@ -4,7 +4,7 @@ import { Header } from '@/components/Header'
 import { Container } from '@/components/Container'
 import { Newsletter } from '@/components/Newsletter'
 import Image5 from "@/images/image5.png"
-import Banner from "@/images/signup.png"
+import Banner from "@/images/1.jpg"
```

Comment:
@renjeroham could rename the image to a more intuitive/meaningful name?

Reply:
@mconyango Hello Mr. Fred, the name change has been committed. Kindly review once again please.